README.md

```diff
@@ -1,8 +1,29 @@
+### ToRoman
+[![Build Status](https://cloud.drone.io/api/badges/nytimes/library/status.svg)](https://cloud.drone.io/nytimes/library) 
```

Comment:
What is this build status?

Reply:
The build status is to show that it works